CreatingReports/CreatingReports.csproj

```diff
@@ -94,6 +95,10 @@
     <Compile Include="Tests\SliderFeature.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Content Include="..\packages\WebDriver.ChromeDriver.win32.85.0.4183.87\content\chromedriver.exe">
```

Comment:
@andreea-qa is it possible for us to just use a Nuget package to manage ChromeDriver just like I did [here](https://github.com/nadvolod/LightPomFrameworkTutorial/blob/56b8bf09b84fcd800759553ff92b6258cea39ed9/ElementInteractions/ElementIdentification.cs#L23) using [WebDriverManger](https://github.com/nadvolod/LightPomFrameworkTutorial/blob/56b8bf09b84fcd800759553ff92b6258cea39ed9/ElementInteractions/packages.config#L11)? Then users don't have to download the old chromedrivers and have any issues.